packages/components/src/side_scroller/index.jsx

```diff
@@ -0,0 +1,136 @@
+import { useEffect, useRef, useState } from 'react';
+import clsx from 'clsx';
+import { size, eventCoordinates, stopEvent } from 'nebenan-helpers/lib/dom';
+import eventproxy from 'nebenan-helpers/lib/eventproxy';
+import styles from './index.module.scss';
+
+import Draggable from '../draggable';
+import Controls from './controls';
+import { DISABLE_SCROLL_DISTANCE } from './constants';
+
+const SideScroller = ({ className: passedClassName, children, ...cleanProps }) => {
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
+  const [canScrollRight, setCanScrollRight] = useState(false);
+  const [height, setHeight] = useState(0);
+
+  const containerRef = useRef(null);
+  const contentRef = useRef(null);
+  const containerWidthRef = useRef(null);
+  const contentWidthRef = useRef(null);
+  const animationIdRef = useRef(null);
+  const startXRef = useRef(null);
+  const startPositionRef = useRef(null);
+  const preventClickRef = useRef(null);
+  const stopListeningToResizeRef = useRef(null);
+
+  const className = clsx(styles.root, passedClassName);
+
+  // Fixes issue with most browsers reducing size of the scrollable element children
+  // as if to compensate for scrollbars, even when they are hidden
+  const style = { height };
+
+  const getScrollableNode = () => containerRef.current;
```

Comment:
Can we inline this one? Please check if there are any usages of `SideScroller` that imperatively called `getScrollableNode` because otherwise I don't really get why we had it in the first place